eva/optimizer/column_mapper.py

```diff
@@ -0,0 +1,97 @@
+
+from eva.planner.function_scan_plan import FunctionScanPlan
+from eva.planner.seq_scan_plan import SeqScanPlan
+from eva.planner.lateral_join_plan import LateralJoinPlan
+from eva.planner.storage_plan import StoragePlan
+from eva.planner.project_plan import ProjectPlan
+from eva.expression.tuple_value_expression import TupleValueExpression
+from eva.expression.constant_value_expression import ConstantValueExpression
+from eva.expression.abstract_expression import AbstractExpression
+from eva.planner.abstract_plan import AbstractPlan
+from eva.utils.single_dispatch_method import singledispatchmethod
+
+
+class ColumnMapper:
+    """
+    Maps the ids for columns
```

Comment:
Elaborate to better explain the role of the file

Reply:
done